config/environments/test.rb

```diff
@@ -43,8 +45,17 @@
   # Print deprecation notices to the stderr.
   config.active_support.deprecation = :stderr
 
+  # Raise exceptions for disallowed deprecations.
+  config.active_support.disallowed_deprecation = :raise
+
+  # Tell Active Support which deprecation messages to disallow.
+  config.active_support.disallowed_deprecation_warnings = []
+
   # Raises error for missing translations.
-  # config.action_view.raise_on_missing_translations = true
+  # config.i18n.raise_on_missing_translations = true
```

Comment:
Oh interesting we use the [i18n-tasks](https://glebm.github.io/i18n-tasks/) gem for this. I think its still worth using the gem, since we get reverse checks (unused translations) and its plugged nicely into our CI.